renovate.json

```diff
@@ -1,6 +1,17 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:recommended"
+    ":dependencyDashboard",
+    ":semanticPrefixFixDepsChoreOthers",
+    "group:monorepos",
+    "group:recommended",
+    "replacements:all",
+    "workarounds:all"
+  ],
+  "packageRules": [
+    {
+      "matchFiles": ["MODULE.bazel"],
```

Comment:
note, this file doesn't strictly need to be in this PR, it could be a separate pre-factoring.

Where possible I'd like to grab from the rules-template (e.g. https://github.com/bazel-contrib/rules-template/blob/main/renovate.json) which is community-maintained "best practices" for rulesets.